packs/spell-effects/spell-effect-fortissimo-composition.json

```
@@ -0,0 +1,75 @@
+{
+    "_id": "rAW5o8MjjuIhYhLg",
+    "img": "systems/pf2e/icons/spells/inspire-heroics.webp",
+    "name": "Spell Effect: Fortissimo Composition",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fortissimo Composition]</p>\n<p>Improve composition bonuses from +1 to +2 or +3 on crit/p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "Critical Success",
+                        "value": "3"
+                    },
+                    {
+                        "label": "Success",
+                        "value": "2"
+                    }
+                ],
+                "flag": "fortissimoBonus",
+                "key": "ChoiceSet",
+                "prompt": "Success type"
```

```suggestion
                "choices": [
                    {
                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
                        "value": "3"
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.success",
                        "value": "2"
                    }
                ],
                "flag": "fortissimoBonus",
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess"
```
We need to use localizations in rule elements to prevent translators from having to dig through the array to translate.